proto/frequenz/api/dispatch/dispatch.proto

```diff
@@ -151,6 +156,7 @@ message DispatchCreateRequest {
   string type = 2;
 
   // The start time
+  // The starting timestamp has to be in the future when creating a dispatch
```

Comment:
I'm not 100% certain we should be defining it like that. What if someone wants to dispatch something NOW?